run.py

```diff
@@ -0,0 +1,7 @@
+import os, subprocess, locale
+cwd = os.getcwd()
+def cmd(execute):
+    return subprocess.call(execute.format(cwd).split())
```

Comment:
Just for straightforward things like this I wouldn't add additional libraries (which may also hide other bugs).